src/resmom/requests.c

```diff
@@ -3895,8 +3895,20 @@ post_chkpt(job *pjob, int  ev)
 		 **	obit is sent.
 		 */
 		if (abort) {
-			exiting_tasks = 1;
-			term_job(pjob);
+			mom_hook_input_t  *hook_input = NULL;
+
+			hook_input = (mom_hook_input_t *)malloc(sizeof(mom_hook_input_t));
```

Comment:
Shouldn't we log an error if the malloc fails? Since post_chkpt() returns void, the caller won't know if anything has gone wrong.

Reply:
@mike0042: I'm not sure how important it is to log the error malloc failure. Sure it will tell the caller that malloc failed, but job would still be terminated. Also, even if the execjob_end hook execute successfully or not, the job would still reach its end point. pbs_mom would not prevent the job from being aborted, given that the checkpoint_abort script already told the job to exit. Let me know if you still feel that I should logged that malloc failure...

Reply:
@bayucan - I see your point. We do log a message from this function if readdir() fails. Besides, if we couldn't malloc a small amount of memory there are big problems happening on the system.